src/Dto/Traits/DtoHelper.php

```diff
@@ -12,8 +12,31 @@ protected function setterByData(array $data): void
             }, $k);
             if (method_exists($this, 'set' . $key)) {
                 $this->{'set' . $key}($v);
+            } else {
+                $key = lcfirst($key);
+                $this->$key = $v;
             }
         }
     }
 
+    protected function getterByAttribute(string $attribute)
+    {
+        $key = preg_replace_callback('/_([a-zA-Z])/', function ($match) {
```

Comment:
Laravel already has helpers translating strings from snake_case to StudlyCase
Str::studly() from Illuminate\Support\Str